apps/prairielearn/src/lib/editors.ts

```diff
@@ -948,6 +954,10 @@ export class QuestionAddEditor extends Editor {
         return { qid: this.qid, title: this.title };
       }
 
+      if (this.isDraft) {
+        return { qid: `draft_${this.draftId}`, title: `draft #${this.draftId}` };
+      }
```

Comment:
Is it necessary for `QuestionAddEditor` to be aware of whether or not something is a draft at all? In my mind, the caller would just specify that `qid` is `__drafts__/draft_...`, and nothing else would have to change (since draft questions are just normal questions with a certain path naming scheme).

Reply:
changed system in a94d9c043b56e640362369ff73f52d0a019803dd.